Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.h

```diff
@@ -34,7 +34,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @protocol ConversationInputTextViewDelegate;
 
-@interface ConversationInputToolbar : UIToolbar
+@interface ConversationInputToolbar : UIView
```

Comment:
It turns out that it's not safe to add subviews to `UIToolbar` in its initializer.  We weren't actually using any `UIToolbar` behavior, so we'll just use a `UIView` instead.

Fixes @michaelkirk 's "can tap on input toolbar on iOS 11" issue.